cmd/asset-syncer/postgresql_utils.go

```diff
@@ -110,13 +110,13 @@ func (m *postgresAssetManager) importCharts(charts []models.Chart, repo models.R
 	return nil
 }
 
-func (m *postgresAssetManager) removeMissingCharts(charts []models.Chart) error {
+func (m *postgresAssetManager) removeMissingCharts(repo models.Repo, charts []models.Chart) error {
 	var chartIDs []string
 	for _, chart := range charts {
 		chartIDs = append(chartIDs, fmt.Sprintf("'%s'", chart.ID))
 	}
 	chartIDsString := strings.Join(chartIDs, ", ")
-	rows, err := m.DB.Query(fmt.Sprintf("DELETE FROM %s WHERE info ->> 'ID' NOT IN (%s) AND info -> 'repo' ->> 'name' = $1", dbutils.ChartTable, chartIDsString), charts[0].Repo.Name)
```

Comment:
When I wrote the test before this change, i found we weren't deleting the files, ever (as there was no cascade delete relationship)... TDD FTW :)